install_config/install/deploy_router.adoc

```diff
@@ -230,6 +230,45 @@ endif::[]
 You can link:../../admin_guide/high_availability.html[set up a highly-available
 router] on your OpenShift cluster using IP failover.
 
+
+[[configuring-router-sharding]]
+=== Configuring Router Sharding
+
+Router sharding is
+link:../../architecture/core_concepts/routes.html#router-sharding[based on labels];
+you set labels on the router
+and make sure there is a corresponding node selector
```

Comment:
This needs some more clarification - its not labels on the router but rather on the routes themselves.
We are basically saying: you can run multiple routers with a route selection criteria that restricts the routers to serving a subset of the routes - a divide and conquer strategy ... taking a set and breaking it up into smaller sets (subsets if you will) and running multiple routers to service those each of those subsets. 

Example a simple naive method would be to alphabetically split routes names starting with A-J get served by router rs1, L-R by router rs2 and the rest by router rs3. 
Or you could even define a custom more complicated method based on SLAs on routes - let's assume you have some routes that have labels on them re: what SLA they need - and let's use high, medium, low as the tag values. We could now select all routes with SLA=high by routers that run on `beefier` (more performant) instances and similarly select low and medium by routers that run on appropriate instances. 
Or you could do this based on namespaces as well (namespace=finance is more critical than namespace=dev ... damn, that was too close - reality h{ur,i}ts!!). 

In essence, a method to slice and dice routes that an individual router (note there could be many pods backing each  router) would service. And if you create these subsets appropriately, you can cover all the routes with different router instances.

Some examples covering the above:
```  
#  The general pool of routers (3)  - where SLA is not high.  
#  Assuming that the routes that need a high SLA have a "SLA=high" label on them, you can run:    
oadm router rone --replicas=0  ...   
oc edit dc/rone  ROUTE_LABELS="SLA != high"     
oc scale dc/rone --replicas=3  
  
# The higher SLA set + run on beefier instances.
oadm router rtwo --selector="zone=beefier-instances" - --replicas=0  ...   
oc edit dc/rtwo  ROUTE_LABELS="SLA=high"    
oc scale dc/rtwo --replicas=7  

#  This set is how to divide the routes based on namespaces.  
#  You could do it based on projects as well. The ones from finance and prod.  
oadm router cthree -replicas=0  ...   
oc edit dc/cthree  NAMESPACE_LABELS="name in (finance, prod)"    
oc scale dc/cthree --replicas=10  

#  The ones from marketing.
oadm router p0 --replicas=0  ...   
oc edit dc/p0 NAMESPACE_LABELS="name=marketing"    
oc scale dc/p0 --replicas=5  

#  The general ones.   
oadm router bb8 --replicas=0  ...   
oc edit dc/bb8  NAMESPACE_LABELS="name not in (finance, prod, marketing)"    
oc scale dc/bb8--replicas=2  

#  Other possibilities include using PROJECT_LABELS, ROUTE_FIELDS and just running 
#  in the scope of a single namespace.  
```   

Hopefully that explained it better.